honeybee_schema/radiance/modifier.py

```diff
@@ -14,7 +14,15 @@ class Void(BaseModel):
 class ModifierBase(IDdRadianceBaseModel):
     """Base class for Radiance Modifiers"""
 
-    modifier: _REFERENCE_UNION_MODIFIERS = Field(
+    type: constr(regex='^ModifierBase$') = 'ModifierBase'
```

Comment:
I don't think we need this `ModifierBase` class anymore if both the modifier and dependency properties are being moved to each of the individual inheriting classes. We can just have all of the modifiers inherit from `IDdRadianceBaseModel`.

Reply:
You're right. We can technically remove it but I kept it there for now for two reasons:

1. Fix this issue in a better way in the future.
2. Have a baseclass for all the Radiance modifiers that @MingboPeng can use for dotnet bindings.

Reply:
Do we need to define the ModifierBase type as a separate type here? If it's acting purely as a class that is being inherited (in the future) won't the child classes always override the type?